### DISCUSSION TITLE
Easy Linux Docker image + instructions for new shibes

### DISCUSSION THREAD

**Discussion by qlpqlp** [dogecoin/dogecoin#2493] [Q&A] [answered]
Hello to all,

I see a lot of new Dogecoin  Docker images flying around GIT, and I think is better to mantain a oficial version of Docker here.

For new shibes that manage serveral servers and DC's, like me, with Docker is easy and it only needs one command do run a full Dogecoin  node without messing the server configurations or stability, like @rnicoll  do with VM's wen streaming :)

Dogecoin still has lower number of nodes usage, because people are scare to mess around, and we have to make it easy and funny for everyone.

I propose a funny setup for Docker image, and will also propose a funny GUI wen installing on Windows or linux. The Dogecoin instalattion Wallet is to "scary" and old, we need a meme GUI :) 
if all agree I can design one, and also build a Docker Image with instructions.

BTW im @inevitable360 on TW

**Comment by AbcSxyZ** [comment, marked as answer]
Hi, I agree, having a docker setup would be great to launch nodes to make dogecoin more accessible. I started Dockerfile draft.

It can launch any binary of the core, including `dogecoind`, with configuration for port, users, whatever. Actually, you have environment variable for each options of the man of each binary *[design to discuss]*.

Take a look at #2337, maybe a good piece of code to try to continue the topic. Help, code review, usage feedback is welcome 🙃

**Comment by usr64525** [comment]
I found these files that could help anyone wanting to setup a VM in VirtualBox.
Now, I don't know how relevant these are because of the learning curve of Linux.
https://github.com/usr64525/usr64525.github.io/tree/main/linux-hosting
